lib/namespaces.js

```diff
@@ -11,7 +11,10 @@ const DEFAULT_RESOURCES = [
   { name: 'replicationcontrollers', Constructor: ReplicationControllers },
   { name: 'pods', Constructor: Pods }
 ].concat([
+  'clusterroles',
```

Comment:
Yeah, but probably out of scope for this PR: https://github.com/godaddy/kubernetes-client/issues/85